tls/main.go

```diff
@@ -620,6 +630,22 @@ func osctrlService() {
 			time.Sleep(time.Duration(_t) * time.Second)
 		}
 	}()
+
+	// Register Prometheus metrics
+	handlers.RegisterMetrics(prometheus.DefaultRegisterer)
+
+	// Creating a new prometheus service
+	prometheusServer := http.NewServeMux()
+	prometheusServer.Handle("/metrics", promhttp.Handler())
+
+	go func() {
```

Comment:
I think the endpoint `/metrics` should be only enabled if the service is configured to do so, not by default.